cumulus/parachains/integration-tests/emulated/networks/rococo-westend-system/src/lib.rs

```diff
@@ -73,9 +73,23 @@ decl_test_bridges! {
 	}
 }
 
+type AssetHubRococoRuntime = <AssetHubRococoPara as Chain>::Runtime;
+type AssetHubWestendRuntime = <AssetHubWestendPara as Chain>::Runtime;
 type BridgeHubRococoRuntime = <BridgeHubRococoPara as Chain>::Runtime;
 type BridgeHubWestendRuntime = <BridgeHubWestendPara as Chain>::Runtime;
 
+pub type AssetHubRococoWestendMessageHandler = BridgeMessagesHandler<
+	AssetHubRococoRuntime,
+	BridgeMessagesInstance3,
+	AssetHubWestendRuntime,
+	BridgeMessagesInstance1,
+>;
+pub type AssetHubWestendRococoMessageHandler = BridgeMessagesHandler<
+	AssetHubWestendRuntime,
+	BridgeMessagesInstance1,
+	AssetHubRococoRuntime,
+	BridgeMessagesInstance3,
+>;
```

Comment:
I think both AH pallets are using Instance1:

```suggestion
	AssetHubRococoRuntime,
	BridgeMessagesInstance1,
	AssetHubWestendRuntime,
	BridgeMessagesInstance1,
>;
pub type AssetHubWestendRococoMessageHandler = BridgeMessagesHandler<
	AssetHubWestendRuntime,
	BridgeMessagesInstance1,
	AssetHubRococoRuntime,
	BridgeMessagesInstance1,
>;
```

AHR: 
```
pub type WithAssetHubWestendMessagesInstance = pallet_bridge_messages::Instance1;
impl pallet_bridge_messages::Config<WithAssetHubWestendMessagesInstance> for Runtime {
```
AHW:
```
pub type WithAssetHubRococoMessagesInstance = pallet_bridge_messages::Instance1;
impl pallet_bridge_messages::Config<WithAssetHubRococoMessagesInstance> for Runtime {
```